apis/Google.Cloud.Spanner.Data/Google.Cloud.Spanner.Data.Tests/PartitionOptionsTests.cs

```diff
@@ -0,0 +1,101 @@
+// Copyright 2023 Google Inc. All Rights Reserved.
+// 
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+// 
+//     http://www.apache.org/licenses/LICENSE-2.0
+// 
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+using Google.Cloud.Spanner.V1;
+using System;
+using System.Threading;
+using Xunit;
+
+namespace Google.Cloud.Spanner.Data.Tests;
+
+public class PartitionOptionsTests
+{
+    [Fact]
+    public void CheckDefaultValues()
+    {
+        var options = PartitionOptions.Default;
+
```

Comment:
Possibly just call `AssertDefaultOptions(PartitionOptions.Default)` for this test?
(If we introduce a new property, it would be good to only have to update one place.)